server/main.lua

```diff
@@ -49,6 +49,12 @@ RegisterNetEvent('qb-vehiclekeys:server:setVehLockState', function(vehNetId, sta
     SetVehicleDoorsLocked(NetworkGetEntityFromNetworkId(vehNetId), state)
 end)
 
+RegisterServerEvent("qb-vehiclekeys:server:synckeys", function(KeyList)
+    if not Config.SaveInDB then return end
+    local Player = QBCore.Functions.GetPlayer(source)
+    Player.Functions.SetMetaData("VehKeys", KeyList)
+end)
+
```

Comment:
```suggestion
```

This is still "insecure" and should be removed
(especially when its not being used anymore)